buildCoreClr.ps1

```diff
@@ -2,6 +2,7 @@
     [switch]$Build,
     [switch]$Uninstall,
     [switch]$Install,
+    [switch]$Docs,
```

Comment:
I cannot see any usage of this switch?

Reply:
yah - that will be a different PR (auto build docs)